dockers/docker-base-bookworm/bin/wrapper_logger.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# This is a wrapper script for logger to take -i option as the default
+# If user provides -id or --id then that option takes precedence
+
+/usr/bin/logger -i "$@"
```

Comment:
Using just `-i` by itself would run into the same problem that we currently have since each call of the wrapper script would have a unique PID. I think we need to call it as 
```
/usr/bin/logger --id="$PPID" "$@"
```
So that the parent script PID is used.

Reply:
@theasianpianist `logger` has two options to specify the PID :- 
1. `-id`  This option specify that the logger will use the PID of the process which invoked the logger command. So two difference process A and B invoking logger will use their respective PID so the rsyslog will have rate limiting fairness for both

2. when you use `--d=$PPID`  two different process A and B will have same PPID which i think is the bash. Using this option is not recommended because it will not be fair for both process A and B